x-pack/plugins/lens/public/xy_visualization/color_assignment.ts

```diff
@@ -111,6 +117,13 @@ export function getAccessorColorConfig(
         triggerIcon: 'disabled',
       };
     }
+    if (layer.layerType === layerTypes.THRESHOLD) {
```

Comment:
nit: could we move it up the line 112? Same as in my previous comment, it's an early return and thanks to that we don't have to execute the lines before (currentYConfig finding)

Reply:
finding `currentYConfig` is still required to return a user defined color for the `yConfig`